s3tests_boto3/functional/test_s3select.py

```diff
@@ -922,6 +1108,70 @@ def test_like_expressions():
 
     s3select_assert_result( res_s3select_like, res_s3select )
 
+@pytest.mark.s3select
+def test_json_like_expressions():
+
+    csv_obj = create_random_csv_object_string(1000,10)
+    json_obj = csv_to_json(csv_obj)
+
+    json_obj_name = get_random_string()
+    bucket_name = get_new_bucket_name()
+
+    upload_object(bucket_name,json_obj_name,json_obj)
+
+    res_s3select_like = remove_xml_tags_from_result(run_s3select_json(bucket_name,json_obj_name,"select count(0) from s3object[*].root where _1.c1 like \"%aeio%\";")).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(run_s3select_json(bucket_name,json_obj_name, "select count(0) from s3object[*].root where substring(_1.c1,11,4) = \"aeio\" ;")).replace("\n","")
+
+    s3select_assert_result( res_s3select_like, res_s3select )
+
+    res_s3select_like = run_s3select_json(bucket_name,json_obj_name,'select  (_1.c1 like "%aeio%") from s3object[*].root ;')
+
+    res_s3select = run_s3select_json(bucket_name,json_obj_name, 'select (substring(_1.c1,11,4) = "aeio") from s3object[*].root ;')
+
+    s3select_assert_result( res_s3select_like, res_s3select )
+
+@pytest.mark.s3select
+def test_json_like_expressions():
```

Comment:
@albin
`test_json_like_expressions` is duplicated name

Reply:
Thanks. Updated the name.